src/App.tsx

```diff
@@ -17,18 +75,31 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                handleSelectChange={handleSelectChange}
+                query={query}
+                handleQueryChange={handleQueryChange}
+                clearQuery={clearQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+
+              {!isLoading && todos.length > 0 && (
+                <TodoList
+                  todo={filteredTodos}
+                  onTodoClick={openModal}
+                  activeTodoId={activeTodoId}
+                  setActiveTodoId={handleClickId}
```

Comment:
The prop 'setActiveTodoId' should be named 'onTodoClick' or similar to indicate that it is an event handler function, following the naming conventions for event handler functions.